<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerPanelClientLoginWindow</class>
 <widget class="QMainWindow" name="ServerPanelClientLoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerPanel - Login</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listServers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtServerAddress">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>391</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUsername">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>170</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPassword">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>170</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblServerAddress">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUsername">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>170</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPassword">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="Line" name="lineServerProperties">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>16</width>
      <height>261</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="lineServerAddress">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>391</width>
      <height>15</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="lineUsername">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>170</width>
      <height>15</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="linePassword">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>170</width>
      <height>15</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="btnAddServer">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="popupMode">
     <enum>QToolButton::InstantPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="btnRemoveServer">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="popupMode">
     <enum>QToolButton::InstantPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
   </widget>
   <widget class="Line" name="lineSaveLogin">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>390</width>
      <height>15</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign-In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
